contrib/babelfishpg_tsql/src/pltsql_utils.c

```diff
@@ -1083,6 +1083,19 @@ update_GrantRoleStmt(Node *n, List *privs, List *roles)
 	stmt->grantee_roles = roles;
 }
 
+void
+update_RevokeRoleStmt(Node *n, List *privs, List *roles)
```

Comment:
Let's reuse update_GrantRoleStmt. We don't need to mark `is_grant = false`, it will be done implicitly while parsing the statement.

Reply:
Done